roles/foreman_provisioning/tasks/main.yml

```diff
@@ -183,44 +200,172 @@
   shell: >
     {{ foreman_provisioning_hammer }} os set-default-template --id {{ foreman_provisioning_centos73.Id }} --config-template-id {{ item.Id }}
   with_items: "{{ kickstart_templates }}"
+# CentOS 7 end
+
+# Fedora 27 start
```

Comment:
Maybe tasks for each os could be in a separate file and we could just include/import them here?